content/en/examples/access/admission-webhook-match-conditions.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+webhooks:
+- name: my-webhook.example.com
+  matchPolicy: Equivalent
+  rules:
+  - operations: ['CREATE','UPDATE']
+    apiGroups: ['*']
+    apiVersions: ['*']
+    resources: ['*']
+  failurePolicy: 'Ignore' # Fail-open (optional)
+  sideEffects: None
+  clientConfig: {} # Omitted for this example
+  matchConditions:
+  - name: 'exclude-leases' # Each match condition must have a unique name
+    expression: '!(request.resource.group == "coordination.k8s.io" && request.resource.resource == "leases")' # Match non-lease resources.
+  - name: 'exclude-kubelet-requests'
+    expression: '!("system:nodes" in request.userInfo.groups)' # Match requests made by non-node users.
+  - name: 'rbac' # Skip RBAC requests, which are handled by the second webhook.
+    expression: 'request.resource.group != "rbac.authorization.k8s.io"'
+# This exmaple illustrates the use of the 'authorizer'. The authorization check is more expensive
```

Comment:
```suggestion
# This example illustrates the use of the 'authorizer'. The authorization check is more expensive
```

I know it's more work, and it _can_ wait 'til beta, but I'd have this as two ValidatingWebhookConfigurations rather one ValidatingWebhookConfiguration defining two webhooks.

I don't see why these two would be paired together, and it's hard to tell (for example: do they have the same `clientConfig`? the elision means we can't see).

As a compromise, how about:
- add a comment before the first webhook (line 4 at time of writing), to explain the overall intent you think it'd have.
- put some vertical white space between the first webhook and the comment that starts the second one. Helps readability.

Reply:
Yeah, I was trying to imply that both would use the same backend. I can add a clientConfig to make it clear, but I'm worried it would just be more clutter. I'm not sure if there is any real difference between grouping multiple webhooks on the same admission configuration vs. multiple objects. IMO the client config should apply to the whole object rather than per-webhook.

I could split the example, but I think that just adds more boilerplate?

Reply:
A comment to the effect that both use the one backend seems enough.

Reply:
No need for an actual `clientConfig` each time.

Reply:
How about yaml anchors? (see latest push)

Reply:
😱 please don't model using yaml anchors in examples